### DISCUSSION TITLE
deprecated namespaces and capstone file not found build warnings

### DISCUSSION THREAD

**Discussion by ghost** [gem5/gem5#936] [Q&A] [answered]
I'm getting these warnings when trying to build gem5. Should I be worried about these, or can I proceed/ignore? Is there a fix?
![Screenshot 2024-03-14 at 8 48 30 PM](https://github.com/gem5/gem5/assets/90482410/bbb53a6f-64c5-4bfb-b793-bc81dca13f05)
The build command I'm using is 'scons build/X86/gem5.opt -j 54'. I'm using gem5 version 23.1 and haven't modified it at all, and I'm running it on X86-64 cpu runninng ubuntu 20.04.6.

**Comment by giactra** [comment, marked as answer]
It's not a problem at all. Capstone is optional; if you don't use it there is no need to include its header. In case you wanted to use Capstone, you would have to install its package and then gem5 would be able to link with it.
